scripts/exglobal_stage_ic.py

```diff
@@ -37,6 +37,7 @@ def main():
     for key in stage.task_config.keys():
         if key.startswith("COM"):
             stage_dict[key] = stage.task_config[key]
+        stage_dict["ENSMEM"] = stage.task_config["ENSMEM"] if "ENSMEM" in stage.task_config else 0
```

Comment:
I realize now that this doesn't do exactly what I intended when I made the suggestion.  If `ENSMEM` is in the `task_config`, then `stage_dict["ENSMEM"]` is set to `task_config["ENSMEM"]` (that much is OK).  But if it isn't in the `task_config`, then `stage_dict["ENSMEM"]` will be set to 0.  I don't think we want this for the GFS system, so this should probably be an if block:

```suggestion
        if "ENSMEM" in stage.task_config:
            stage_dict["ENSMEM"] = stage.task_config["ENSMEM"]
```